torch/fx/_symbolic_trace.py

```diff
@@ -849,6 +849,18 @@ def wrapped(*args, **kwargs):
             )
             return_proxy.node.meta["is_wrapped"] = True
             return return_proxy
+        
+        # import here to avoid circular imports
+        from .experimental.proxy_tensor import get_innermost_proxy_mode, proxy_call
+
+        # If there is no input with proxy, see if we are proxying fake tensors
+        proxy_mode = get_innermost_proxy_mode()
```

Comment:
This is probably not sufficient, there is such a thing as nested make_fx calls, so let's use `with disable_proxy_modes_tracing()` insted`

Reply:
Let's also make a test for nested make_fx calls to show this fails

Reply:
https://github.com/pytorch/pytorch/pull/93290 should orthogonally fix this, but let's still just disable the entire proxy tensor dispatch mode stack to be safe...